doc/DockerBuild.md

```diff
@@ -1,6 +1,3 @@
-By default, the [Helm Charts](https://github.com/vitessio/vitess/tree/main/helm)
-point to the `vitess/lite` image on [Docker Hub](https://hub.docker.com/u/vitess/).
-
```

Comment:
Haha, I never knew this was here. The helm charts only used `lite` at the very very beginning before k8s added stateful sets. The completed charts always used the individual binaries.